source/_UIAHandler.py

```diff
@@ -140,7 +142,7 @@
 
 UIAEventIdsToNVDAEventNames={
 	UIA_LiveRegionChangedEventId:"liveRegionChange",
-	#UIA_Text_TextChangedEventId:"textChanged",
+	UIA_Text_TextChangedEventId:"textChanged",

```

Comment:
Shouldn't this be:

```suggestion
	UIA_Text_TextChangedEventId:"textChange",
```

This makes the event_textChanged definition in UIAConsole obsolete.

Alternatively, if you don't want to map this to the textChange event, it might be better to prefix the event name with UIA, i.e:

```suggestion
	UIA_Text_TextChangedEventId:"UIA_textChanged",
```

Reply:
This line was commented out, and I didn't want to change it (thought it was called `textChanged` for a reason). I agree that it's confusing though!

A quick scan of the NVDA source shows that `textChange` is used more than `textChanged` (in fact, `textChanged` is basically only used for UIA). Surely this must've been a mistake.